### ISSUE TITLE
Database issue after newest update

### ISSUE THREAD

**Issue by Jay12134** [suriyun-production/mmorpg-kit-docs#1820]
Hello, When I updated to the newest version players are unable to login to old accounts only can make new account.

I noticed lots of errors in yellow on console do you know why this might be happening? everything is the same and all passwords are right. 

Thank you for your help!

![1057b5882a82c442d21e81d8122b087f](https://user-images.githubusercontent.com/121772617/210186134-c3d6f562-6772-47ff-a66b-b865f3ad1f8c.png)

![19a3799c8efa8ad51b42bbc82b948d19](https://user-images.githubusercontent.com/121772617/210186132-794bba36-0e59-46ac-b0a3-f97105423b77.png)

**Comment by insthync** [comment]
Those warning may be shown because you've set up for client build, so server codes will be excluded.

**Comment by Jay12134** [comment]
Thanks for your response.

I updated with the same method as in the document to delete some folders and update. I did nothing to the database and also didn't overwrite my scenes. 

Was there maybe something new I need to add to my scenes? Very weird that the database didn't connect after updating. I'm using MySQL. I've updated before it's not my first time but was wondering if maybe something changed with database that I needed to do.

Thanks!

**Comment by insthync** [comment]
May I see your mySqlConfig.json ?

**Comment by Jay12134** [comment]
I don't use a config file as it never worked for me, I use the inspector in the network scene.

**Comment by insthync** [comment]
Then when you change some settings you have to rebuild and rerun the server.

**Comment by Jay12134** [comment]
Alright I'll try again, last I checked all the settings are correct since it worked before the last update I didn't change any settings. 

I just wanted to check and make sure I didn't need to do anything with the new update and MySQL. I just updated code was wondering if I needed to do anything with the database or init scene.

Thanks

**Comment by Jay12134** [comment]
`{"mySqlAddress":"127.0.0.1","mySqlPort":3306,"mySqlUsername":"admin","mySqlPassword":"changedpass","mySqlDbName":"mmorpg_kit"}`

^so I tried to go back to the old version I was using and still a problem, I cant connect to the mysql database when I connected just fine no problem before. All the settings are the same and match exact. 

This is so weird because I don't see anything in the boot console about mysql even though its set to mysql, I've rebuilt server too. I am so lost with this it was working perfect for 8 months then all of a sudden when I updated to newest it wont connect to any mysql it just falls back on sqlite.  :(

sorry to be a pain in the ass, but you are my only hope for help on this.
